README.md

```diff
@@ -388,6 +388,18 @@ The launcher for a JWST analysis is also located in `pastis/launchers`, as `run_
 ### HiCAT
 The HiCAT simulator is private and its support is only provided for internal use.
 
+HiCAT-PASTIS compatible conda env:
```

Comment:
You rebased on `develop` rather than `origin/develop` while your develop branch was out of date, which is why you're in fact not yet up to date after the rebase. Please fix that before we proceed.

Reply:
@ananyasahoo1904 here